<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMetaWindow</class>
 <widget class="QDialog" name="TestMetaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод мета-информации теста</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="subjectLabel">
       <property name="text">
        <string>Предмет</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="subjectEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="themeLabel">
       <property name="text">
        <string>Тема</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="themeEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Автор</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="authorEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Почта</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="emailEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="createBtn">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
